addons/test_mail/tests/test_mail_activity.py

```diff
@@ -159,6 +159,33 @@ def _employee_crash(records, operation):
             self.assertEqual(1, read_group_result[0]['summary_count'])
             self.assertEqual('Summary', read_group_result[0]['summary'])
 
+    def test_activity_action_open_document_no_access(self):
```

Comment:
Should be defined as first method in class, as tests are executed alphabetically, better order test methods alphabetically also :) that way execution order follows definition order, easier to find and fix when having several crashing tests :)